pydm/pydm_ui.py

```diff
@@ -7,7 +7,7 @@
 # WARNING! All changes made in this file will be lost!
 
 
-from qtpy import QtCore, QtGui, QtWidgets
+from PyQt5 import QtCore, QtGui, QtWidgets
```

Comment:
I would recommend reverting this back to qtpy to keep in line with the rest of pydm